patches/aesthetics.py

```diff
@@ -261,7 +261,53 @@ def reduceMessageLengths(rom, rnd):
         "#####, how do I download RAM?",
         "Is this a delayed April Fool's Joke?",
         "Dayoman is close to getting 120% in 1:20!",
-        "Play as if your footage will go in a\nSummoning Salt video."
+        "Play as if your footage will go in a\nSummoning Salt video.",
+        "I hope you prepared for our date later.",
+        "Isn't this the game where you date a seagull?",
+        "!tele broken, pls fix",
+        "You look pretty good for a guy who probably drowned.",
+        "I heard Matty's a cheater.",
+        "Remember, we race on Sundays.",
+        "Pink Batman delayed this race for lunch.",
+        "This randomizer was made possible by players like you. \n \n Thank you!",
+        "Now with real fake doors!",

```

Comment:
I think this was supposed to be "floors" ("real fake floors" is a thing Matty says when the Eagle quick kill is possible)